vendor/assets/javascripts/parallax.js

```diff
@@ -0,0 +1,58 @@
+(function ($) {
+
+    $.fn.parallax = function () {
+      var window_width = $(window).width();
+      // Parallax Scripts
+      return this.each(function(i) {
+        var $this = $(this);
+        $this.addClass('parallax');
+
+        function updateParallax(initial) {
+          var container_height;
+          if (window_width < 601) {
+            container_height = ($this.height() > 0) ? $this.height() : $this.children("img").height();
```

Comment:
Line is too long.<br>Identifier 'container_height' is not in camel case.<br>Mixed double and single quotes.